src/include/cart/api.h

```diff
@@ -423,6 +424,19 @@ crt_reply_get(crt_rpc_t *rpc)
 uint64_t
 crt_hlc_get(void);
 
+/**
+ * Return the second timestamp of hlc.
+ *
+ * \param[in] hlc              HLC timestamp
+ *
+ * \return                     The timestamp in second
+ */
+static inline uint64_t
+crt_hlc2sec(uint64_t hlc)
+{
+	return hlc / NSEC_PER_SEC;
```

Comment:
I'd be better use the following to be more correct:
return (hlc & ~CRT_HLC_MASK) / NSEC_PER_SEC;